t/unit/contrib/test_pytest.py

```diff
@@ -32,3 +34,37 @@ def test_noop():
         result.stdout.fnmatch_lines_random(
             "*PytestUnknownMarkWarning: Unknown pytest.mark.celery*"
         )
+
+
+@pytest.fixture
+def fixed_import(mocker):
+    def import_module(module, package=None):
```

Comment:
Why are we monkeypatching here if the code of `import_module()` is exactly the same as the monkey patched version?

Reply:
In the first place, I wanted to show that the issue occurs and this is why I added these test cases. I will extract the failing tests to the issue.